modules/wear_detect.js

```diff
@@ -11,9 +11,7 @@ exports.isWorn = function() {
   return new Promise(resolve => {
     if (Bangle.isCharging())
         return resolve(false);
```

Comment:
Nice idea to make a library.

I'd like to suggest to add a check for the global sleeplog object, as the [sleeplog app](https://banglejs.com/apps/?id=sleeplog) is tracking the wearing status too:
```
// sleeplog.status values:
// undefined = service stopped,  0 = unknown, 1 = not worn, 2 = awake, 3 = light sleep, 4 = deep sleep
if (global.sleeplog) {
  if (sleeplog.status) return resolve(sleeplog.status > 1 ? true : false);
}
```

Reply:
Sounds like a good plan. I'll merge this PR but we can add another one with this in